### PR TITLE
chore: add pvc support to couchbase helm

### PR DESCRIPTION
**Pull request by psawantdrest**

### CODE REVIEW

File: charts/couchbase/templates/pvc.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{{- if .Values.pvc.create -}}
```
Comment (SamTealeDrest) [inline charts/couchbase/templates/pvc.yaml:1]: Can we make this persistence.enabled format to match most of upstream charts?

File: charts/couchbase/templates/pvc.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  name: {{ include "couchbase.fullname" . }}
+  labels:
+    {{- include "couchbase.labels" . | nindent 4 }}
+spec: 
+  accessModes: 
+    - {{ .Values.pvc.accessMode }}
+  resources: 
+    requests: 
+      storage: {{ .Values.pvc.size }}
+  storageClassName: {{ .Values.pvc.storageClass }}
```
Comment (SamTealeDrest) [inline charts/couchbase/templates/pvc.yaml:14]: This should be wrapped in an if and not set if empty so k8s will use default storage class

**Review by SamTealeDrest — CHANGES_REQUESTED** [review]
Looks good, a few bits mentioned inline. We will also need a volume mount and volume declaring in the deployment.yaml template

**Comment by psawantdrest** [comment]
> Looks good, a few bits mentioned inline. We will also need a volume mount and volume declaring in the deployment.yaml template

What should be the values for mountPath and name under volumeMounts

File: charts/couchbase/templates/pvc.yaml (hunk 1 above)
Comment (psawantdrest) [reply charts/couchbase/templates/pvc.yaml]: modified it to enabled

**Review by SamTealeDrest — APPROVED** [review]

**Pull request merged**